measCompApp/src/measCompAppInclude.dbd

```diff
@@ -1,4 +1,6 @@
 include "base.dbd"
+include "PVAServerRegister.dbd"
+include "qsrv.dbd"
```

Comment:
Remove these lines, they are not needed for test IOC and won't work on older versions of base.